src/cmatrix.cpp

```diff
@@ -62,417 +62,939 @@
 
 #include <stdlib.h>
 #include <string.h>
-#include <tiffio.h>
 
+#include <ome/files/in/OMETIFFReader.h>
+#include <ome/files/FormatReader.h>
+#include <ome/files/in/TIFFReader.h>
+#include <ome/files/tiff/IFD.h>
 
-using namespace std;
-//-----------------------------------------------------------------------
+#include <tiffio.h>
+//using namespace ome::files::tiff; //Has conflicts with #include <tiffio.h>
 
+//value of 1: using OMELibrary to read tiled tiff formats
+//Any other value: using libtiff to read tiled tiff formats
+#define useOMELibrary 1
```

Comment:
Why is this set here as a define? If ome is required to build wnd-charm (which it is), then a user shouldn't have to change the source code and recompile their code in order to use it. Instead consider a global (extern) which has a function that can set it, and can be called from a command line argument or python library.

Reply:
This section is no longer needed per our decision to remove OME Library